problems/removeNumbersAtOddIndices.js

```diff
@@ -3,7 +3,11 @@
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
 function removeNumbersAtOddIndices() {
-
-}
+    let arr = []
+    for(let i = 0; arr.length; i = i++)
+    if(i % 2 == 0) {
+        arr.splice(i, 0);
```

Comment:
Generally we want to avoid mutating an array while iterating through it. Try this problem again with .filter.